app-backup/borgbackup/borgbackup-1.2.0.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Deduplicating backup program with compression and authenticated encryption"
+DESCRIPTION="BorgBackup (short: Borg) is a deduplicating backup program.
+Optionally, it supports compression and authenticated encryption.
+
+The main goal of Borg is to provide an efficient and secure way to backup data.
+The data deduplication technique used makes Borg suitable for daily backups
+since only changes are stored.
+The authenticated encryption technique makes it suitable for backups to not
+fully trusted targets."
+HOMEPAGE="https://github.com/borgbackup/borg"
+COPYRIGHT="201-2021 Mathias Vogelgesang"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="7659322a078dd3705f89345d0b18b4cdb2d7fee1b65391b0596cb75a19cac64f"
+SOURCE_FILENAME="borgbackup-$portVersion.tar.gz"
+SOURCE_DIR="borg-$portVersion"
+SOURCE_URI_2="https://github.com/hynek/argon2-cffi/archive/refs/tags/21.3.0.tar.gz"
+CHECKSUM_SHA256_2="1f5cdc94200eccdf8c63c1cc5c0c72e38076d8e81968218b0ec0573fd72b2ba4"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	$portName = $portVersion
+	cmd:borg
+	cmd:borgfs
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	argon2_cffi_bindings${secondaryArchSuffix}_python39
+	cffi${secondaryArchSuffix}_python39
+	msgpack_python39
+	packaging_python39
+	pycparser_python39
+	pyparsing_python39
+	lib:libargon2$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libdeflate$secondaryArchSuffix
+	lib:libffi$secondaryArchSuffix
+	lib:liblz4$secondaryArchSuffix
+	lib:libxxhash$secondaryArchSuffix
+	lib:libzstd$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	devel:libdeflate$secondaryArchSuffix
+	devel:liblz4$secondaryArchSuffix
+	devel:libxxhash$secondaryArchSuffix
+	devel:libzstd$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	python_pkgconfig_python39
+	setuptools_scm_python39
+	wheel_python39
+	cmd:gcc$secondaryArchSuffix
+	cmd:cython3.9$secondaryArchSuffix
+	cmd:git
+	cmd:pip3.9
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python3.9
+	"
+
+SETUPTOOLS_SCM_PRETEND_VERSION=$portVersion
+export SETUPTOOLS_SCM_PRETEND_VERSION
```

Comment:
@ThomasWaldmann this did the trick

Reply:
guess that is a slightly dirty workaround around the real root cause of this issue.

Reply:
One that works, whit a full clone and build in Terminal I get `1.3.0a1.post4+ga466d7` as version in _version.py

Reply:
the you did not checkout the 1.2.0 tag.
or there is something dirty in the workdir.

Reply:
(that rather looks like master branch, which is using 1.3.0+ now, not 1.2-maint branch nor 1.2.0 tag)

Reply:
master branch yes, this one uses the latest release archive